README.md

```diff
@@ -17,6 +17,15 @@ Kuzzle enables you to build modern web applications and complex IoT networks in
 * **User Management**: login, logout and security rules are no more a burden.
 * **Extensible**: fit Kuzzle to your needs by leveraging the plugin system.
 
+## Kuzzle in production
+
+Kuzzle is production-proof, your applications are [deployed securely in a robust environment](https://kuzzle.io/products/by-features/on-premises/). 
```

Comment:
The page you're referencing to explains that Kuzzle can be deployed on anyone environment... I would change that sentence slighlyt:

```suggestion
Kuzzle is production-proof, and can be [deployed anywhere](https://kuzzle.io/products/by-features/on-premises/). 
```